charts/posthog/templates/cronjob.yaml

```diff
@@ -0,0 +1,72 @@
+{{- if .Values.clickhouse.backup.enabled }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: clickhouse-backup-cron
+spec:
+  # every night
+  schedule: {{ .Values.clickhouse.backup.backup_schedule }}
+  concurrencyPolicy: "Forbid"
+  jobTemplate:
+    spec:
+      backoffLimit: 1
+      completions: 1
+      parallelism: 1
+      template:
+        spec:
+          restartPolicy: Never
+          containers:
+            - name: run-backup-cron
+              image: yandex/clickhouse-client:latest
```

Comment:
Can we make this configurable and, as default, use the same version used for `clickhouse-server`